src/modules/UI/components/MaxButton/styles.js

```diff
@@ -1,21 +1,22 @@
 // @flow
+import { THEME } from '../../../../theme/variables/airbitz.js'
```

Comment:
This whole MaxButton component isn't used, so we don't need any of these changes; just delete the whole thing.

Reply:
fixed